mne/viz/_brain/_brain.py

```diff
@@ -80,6 +80,109 @@ class _Brain(object):
         A dictionary of pysurfer.Surface objects for each hemisphere.
     overlays : dict
         The overlays.
+
+    Notes
+    -----
+    This table shows the capabilities of each Brain backend ("✓" for full
+    support, and "-" for partial support):
+
+    .. table::
+       :widths: auto
+
+    +--------------------------------+--------------+-----------------------+
```

Comment:
Doesn't this need to be indented three more spaces to be part of the above `table` directive?